style.css

```diff
@@ -188,55 +182,69 @@ a {
   flex-grow: 0;
 }
 
-#main-works {
+.works_section {
   display: grid;
-  justify-content: center;
   width: 375px;
+  justify-items: center;
+  padding-top: 20%;
+  padding-bottom: 10%;
+  row-gap: 100px;
 }
```

Comment:
- [ ] Kinsly you may also consider sticking to the card style as much as possible as the one provided in Figma
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/196739458-31f5f232-3a98-4088-809d-1309aa6bb0b3.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/196739742-31aaad64-dede-4d42-a1bf-4933cc1fbfd2.png)

As you can see how your `button` is overlapping on the card